torch/csrc/jit/codegen/cuda/index_compute.cpp

```diff
@@ -669,13 +671,80 @@ void IndexCompute::run(const LoopIndexing& loop_indexing) {
     }
   }
 
+  // Resolve the index vals that could be resolved with only
```

Comment:
Is this supposed to be `that could only be resolved with the loops...`?

Reply:
What you're saying is you're resolving anything you can with the loops that consumer_tv doesn't share
What I'm saying is you're resolving loops that cannot be resolved without the loops that consumer_tv doesn't share with its consumers

Reply:
Yes. Both `only with` and `with only` are true here. Will think about formalizing in follow ups.